File: docs-site/android/history.kt
import java.time.Instant
import java.time.temporal.ChronoUnit

val now = Instant.now()

// Set start date to 4 days ago
val startDate: Long = now.minus(4, ChronoUnit.DAYS).toEpochMilli()

// Set end date to 2 days ago 
val endDate: Long = now.minus(2, ChronoUnit.DAYS).toEpochMilli()

val history = client.history("chat.room1", start, end)

// OR

// This will return messages from $start to now()
val history = client.history("chat.room1", start, null)

File: docs-site/python/history.py
from datetime import datetime, timedelta, UTC

now = datetime.now(UTC)

# Set start date to 4 days ago
start = now - timedelta(days=4)

# Set end date to 2 days ago 
end = now - timedelta(days=2)

history = await client.history("chat.room1", start, end)

# OR

# This will return messages from $start to now()
history = await client.history("chat.room1", start)

# --------------------------------------------------------
# $history will look like
[
    {
        id: "<MESSAGE ID>",
        topic: "<TOPIC MATCHING TOPIC / WILDCARD TOPIC>",
        message: <Actual message as string, number or dict>,
        timestamp: "<Timestamp at which message was sent>"
    }, 
    ...
]